### PR TITLE
Dev/345 modify dashboard

### PR DESCRIPTION
**Pull request by ta-chibana**
# 概要
dashboardに
- グループ別の最新月報（とりあえず各グループ10件）
- 未登録タグ全件

# 再現・確認手順
adminアカウントでログインして `/admin` へ

# 対応Issue（任意）
#345

# ToDo

- [x] ラベル付け
- [x] Assigneesで自分を選択

### CODE REVIEW

File: app/admin/dashboard.rb [hunk 1]
```diff
@@ -1,32 +1,43 @@
...
-    div class: 'blank_slate_container', id: 'dashboard_default_message' do
-      span class: 'blank_slate' do
-        span I18n.t('active_admin.dashboard_welcome.welcome')
-        small I18n.t('active_admin.dashboard_welcome.call_to_action')
+    columns do
+      Group.active.each do |group|
+        column max_width: '45%', min_width: '45%' do
+          panel "最新の月報 【#{group.name}グループ】" do
+            reports = MonthlyReport
+              .includes(:user)
```
Comment (freezer530000) [inline app/admin/dashboard.rb]: __[rubocop]__
* Align the operands of an expression in an assignment spanning multiple lines.

File: app/admin/dashboard.rb [hunk 2]
```diff
@@ -1,32 +1,43 @@
...
+              .where(user_id: group.users)
```
Comment (freezer530000) [inline app/admin/dashboard.rb]: __[rubocop]__
* Align the operands of an expression in an assignment spanning multiple lines.

File: app/admin/dashboard.rb [hunk 3]
```diff
@@ -1,32 +1,43 @@
...
+              .order(created_at: :desc)
```
Comment (freezer530000) [inline app/admin/dashboard.rb]: __[rubocop]__
* Align the operands of an expression in an assignment spanning multiple lines.

File: app/admin/dashboard.rb [hunk 4]
```diff
@@ -1,32 +1,43 @@
...
+              .limit(DISPLAY_MONTHLY_REPORT_LIMIT)
```
Comment (freezer530000) [inline app/admin/dashboard.rb]: __[rubocop]__
* Align the operands of an expression in an assignment spanning multiple lines.

**Comment by freezer530000** [comment]
Click here if you want to create rubocop's auto-correct Pull Request
[Auto Correct by SideCI](https://sideci.com/gh/42654368/issues/pull_request/399/diff/ef01c282b960bf04b2c7efeb17ea3db65b4dadca?open_modal=1#rubocop)

See the documentation if you want to get more information about auto-correct.
[Document](https://sideci.com/ja/docs/autofix-rubocop)

**Comment by bakunyo** [comment]
LGTM

**Comment by t-tonchim** [comment]
👍

**Pull request merged**